# 局部地图
key_frame_distance: 1.0 # 关键帧距离
local_frame_num: 20

# 各配置选项对应参数
## 匹配相关参数
res : 1.0
step_size : 0.1
trans_eps : 0.01
max_iter : 30
## Voxel滤波相关参数
local_map:
    leaf_size: [0.6, 0.6, 0.6]
frame:
    # leaf_size: [1.3, 1.3, 1.3]
    leaf_size: [0.6, 0.6, 0.6]
